core/server/src/main/java/alluxio/master/file/FileSystemMasterClientServiceHandler.java

```diff
@@ -147,7 +147,13 @@ public FileInfo getStatusInternal(long fileId) throws AlluxioTException {
     }
   }
 
+  /**
+   * {@inheritDoc}
+   *
+   * @deprecated since version 1.1 and will be removed in version 2.0
```

Comment:
if a URI is known then the underFS address can be inferred from the UFS path stored in the result of getStatus(uri)